WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -1113,7 +1127,10 @@ d_juhuria = {
 	culture = juhuro
 	
 	allow = {
-		always = no
+ 		OR = {
+			culture = juhuro
+			religion = tzadikite
+		}
```

Comment:
Because it is titular, it is better that they create a dejure duchy or get a dynamic dejure structure,